docs/how-to-guides/run-and-evaluate-a-flow.md

```diff
@@ -108,9 +108,7 @@ In this guide, we use [eval-classification-accuracy](https://github.com/microsof
 
 After the run is finished, you can evaluate the run with below command, compared with the normal run create command, note there are two extra arguments:
```

Comment:
Column-mapping may not be an argument specially for evaluation. We may need also introduce column-mapping in the above section "Create a batch run".

Reply:
Good point, but seems section "Create a batch run" didn't use `column-mapping` argument. Since this is just a quick start doc, maybe we can just leave arguments introduction for `pf.run` in reference doc?

Reply:
Offline synced, will add this in newly added doc,